net/sched/sch_skbprio.c

```diff
@@ -192,12 +176,11 @@ static int skbprio_change(struct Qdisc *sch, struct nlattr *opt,
 	const unsigned int min_limit = 1;
 
 	if (ctl->limit == (typeof(ctl->limit))-1)
-		q->max_limit = max(qdisc_dev(sch)->tx_queue_len, min_limit);
-	else if (ctl->limit < min_limit ||
-		ctl->limit > qdisc_dev(sch)->tx_queue_len)
+		return max(qdisc_dev(sch)->tx_queue_len, min_limit);
```

Comment:
Could replace `return` with `sch->limit =`?

Funny, but it's the second time I point this up (see previous round of review).

Reply:
Sorry about that! Will be more careful.